seqr/views/apis/project_api.py

```diff
@@ -314,15 +341,13 @@ def _add_tag_type_counts(project, project_variant_tags):
         'description': '',
         'color': 'grey',
         'order': 100,
-        'numTags': SavedVariant.objects.filter(family__project=project, variantnote__isnull=False).distinct().count(),
+        'numTags': project_notes.aggregate(count=Count('saved_variants__guid', distinct=True))['count'],
```

Comment:
this is getting at the same data (number of distinct saved variant rows associated with at least one variant note), but the way it was constructed before was running a full subquery on the saved variant table and actually getting all the records and then counting those, whereas this just runs a `COUNT` query on the database and does not get any records, which is much faster